lib/install/inflate-shrinkwrap.js

```diff
@@ -84,6 +84,7 @@ function tarballToVersion (name, tb) {
   } else {
     matchRegTarball = new RegExp(`^${registry}(.*)?/-/\\1-(.*)[.]tgz$`)
   }
+  if (!tb) return
```

Comment:
nit: @isaacs what is this for?

Reply:
That seems... unrelated to this commit.  Must be something I was testing out?  I'll back it out.